### ISSUE TITLE
Firefox on Mobile Support

### ISSUE THREAD

**Issue by aneesh-neelam** [karthikb351/AutoCaptcha-for-Chrome#6]
Could you make the Firefox Add-on such that it'll work on Firefox for Android and Firefox OS too? 

I think it's just a small configuration change to existing Firefox Add-on, I'm not sure.

**Comment by karthikb351** [comment]
Yeah, I forgot about this altogether. Will do it over the next day or so. It's just repackaging it.
